inflate.c

```diff
@@ -10,6 +10,7 @@
 #include "inflate_p.h"
 #include "inffixed_tbl.h"
 #include "functable.h"
+#include <stdbool.h>
```

Comment:
@Dead2 we okay using this include? First in the library. I had mentioned it before.

Reply:
Don't think we need this, I went with your latter suggested approach which rendered it unnecessary.